### PR TITLE
New custom domain service

### PR DESCRIPTION
**Pull request by abbeykos**
Standalone update about custom domain service. Contains localhost URLs.

### CODE REVIEW

File: content/updates/2018-05-21-new-custom-domain-service.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
++++ 
+date = "2018-05-21" 
+title = "New custom domain service" 
++++
+
+### New custom domain service
+For custom domain support, cloud.gov provides a [CDN service](https://cloud.gov/docs/services/cdn-route/) that uses AWS CloudFront. AWS CloudFront is outside the AWS FedRAMP P-ATO boundary, so we’ve updated the CDN service documentation to [explain the compliance impact of using this service more clearly](http://localhost:1313/docs/services/cdn-route/#before-you-use-this-service).
```
Comment (mogul) [inline content/updates/2018-05-21-new-custom-domain-service.md]: Needs real/non-localhost URL

File: content/updates/2018-05-21-new-custom-domain-service.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+cloud.gov now also provides a [custom domain service](http://localhost:1313/docs/services/custom-domains/) without CloudFront. You can use the CDN service or custom domain service according to your technical and compliance needs. If you’re using the CDN service and you need to switch to the custom domain service, [follow the instructions on the custom domain page](http://localhost:1313/docs/services/custom-domains/).
```
Comment (mogul) [inline content/updates/2018-05-21-new-custom-domain-service.md]: Needs real, non-localhost URL

**Review by mogul — CHANGES_REQUESTED** [review]
Going to have to fix these URLs, but they're not really available until https://github.com/18F/cg-site/pull/1259 is merged, which itself is not mergeable without the service itself made available. I'm changing the subject line to say [WIP] as an explicit signal that we can't merge this one yet.

File: content/updates/2018-05-21-new-custom-domain-service.md (hunk 1 above)
Comment (brittag) [reply content/updates/2018-05-21-new-custom-domain-service.md]: More info - we try to use this relative URL format, which makes it easier to keep the links working (you'll get a build error if a link is broken in a PR, so you know to fix it before merging and deploying the change) and also makes links work as expected when running the site locally: https://github.com/18F/cg-site#links

**Comment by pburkholder** [comment]
This fails CI with:

```
25lBuilding sites … ERROR 2018/05/22 18:02:21 error processing shortcode "_internal/shortcodes/relref.html" for page "updates/2018-05-21-new-custom-domain-service.md": template: _internal/shortcodes/relref.html:1:76: executing "_internal/shortcodes/relref.html" at <relref .Page (.Get 0...>: error calling relref: No page found with path or logical name "docs/services/cdn-route/".
ERROR 2018/05/22 18:02:21 error processing shortcode "_internal/shortcodes/relref.html" for page "updates/2018-05-21-new-custom-domain-service.md": template: _internal/shortcodes/relref.html:1:76: executing "_internal/shortcodes/relref.html" at <relref .Page (.Get 0...>: error calling relref: No page found with path or logical name "docs/services/cdn-route/".
ERROR 2018/05/22 18:02:21 error processing shortcode "_internal/shortcodes/relref.html" for page "updates/2018-05-21-new-custom-domain-service.md": template: _internal/shortcodes/relref.html:1:76: executing "_internal/shortcodes/relref.html" at <relref .Page (.Get 0...>: error calling relref: No page found with path or logical name "docs/services/custom-domains/".
ERROR 2018/05/22 18:02:21 error processing shortcode "_internal/shortcodes/relref.html" for page "updates/2018-05-21-new-custom-domain-service.md": template: _internal/shortcodes/relref.html:1:76: executing "_internal/shortcodes/relref.html" at <relref .Page (.Get 0...>: error calling relref: No page found with path or logical name "docs/services/custom-domains/".
Total in 345 ms
Error: Error building site: logged 4 error(s)
```

because the relrefs need to be to `.md` files.  

@abbeykos At this point you can't see the CI failures, sorry, that's a current limitation we're working under with our CI system, but we should be able to have you run `hugo` locally and see these errors before they get to CI. LMK if you want to pair on setting that up.

**Comment by jontours** [comment]
Yeah, it's probably the use of `relref` that is being problematic.

**Review by mogul — APPROVED** [review]

**Pull request merged**